helpers/upgradable-contracts.js

```diff
@@ -39,7 +39,7 @@ export async function setupEtherRouter(interfaceContract, deployedImplementation
     const fName = value.name;
     const fType = value.type;
     // These are from DSAuth, and so are on EtherRouter itself without any more help.
-    if (fName !== "authority" && fName !== "owner") {
+    if (fName !== "authority" && fName !== "owner" && !fName.includes("coverage_")) {
```

Comment:
Instrumentation is now done using an injected public function (instead of an event) and needs to be excluded here.